application/templates/static_site/measure.html

```diff
@@ -410,6 +410,12 @@ <h3 class="heading-small">
           <div itemprop="spatialCoverage" itemscope itemtype="http://schema.org/Place">
             <meta itemprop="name" content="{{ measure_version.format_area_covered() }}">
           </div>
+          {% if measure_version.published_at %}<meta itemprop="dateModified" content="{{ measure_version.published_at }}">{% endif %}
```

Comment:
Should we have a custom formatter to ensure we have ISO 8601 format here?  Or are we 100% confident that Jinja2/Python will give us that by default?

Reply:
@TheDoubleK it seems to right now, but I have no idea if that's guaranteed or not..? 🤷‍♂️

Reply:
Python doesn't have good support for ISO8601, amazingly. They're similar, but I think they lack the timezone information that forms a valid ISO8601. Easiest way to do this is probably add a third-party lib like iso8601 and make a new filter based off that. Maybe?

Reply:
Time zones aren't needed for dates though? (we only seem to store the date it was published, not the actual time...)

Reply:
Ah yes, published_at is a date-only field, so I guess this would be OK as-is.  But still to be sure we could define our own filter something like 
```
def format_iso8601_date(date):
    if date is None:
        return ""
    return date.strftime("%Y-%m-%d")
```

Reply:
Good idea, done. 👍